src/v0/destinations/rockerbox/data/RockerboxTrackConfig.json

```diff
@@ -1,14 +1,20 @@
 [
   {
-    "sourceKeys": "userId",
+    "sourceKeys": "userIdOnly",
     "destKey": "customer_id",
     "required": true,
```

Comment:
Not make it a required field. Otherwise, all events without anonymousId will be failed.

Reply:
Updated